docs/culture/psychological-safety.md

```diff
@@ -0,0 +1,107 @@
+# Psychological Safety
+
+One of the primary reasons we host hackweeks is to help build a culture of collaboration. We do this because research is becoming increasingly interdependent, and because we know that cross-disciplinary exchanges increase the likelihood for us to address complex challenges. We also think that collaboration is fun and keeps our work more interesting and engaging!
```

Comment:
```suggestion
One of the primary reasons we host hackweeks is to help build a culture of collaboration. We do this because research is becoming increasingly interdependent and because we know that cross-disciplinary exchanges increase the likelihood for us to address complex challenges. We also think that collaboration is fun and keeps our work more interesting and engaging!
```